web/themes/custom/cloudy/templates/layout/page.html.twig

```diff
@@ -184,7 +184,6 @@
                 {% include "@components/contact-item/contact-item.twig" with {
                   "email": 'cityinfo@portlandoregon.gov',
                   "phone": '503-823-4000',
-                  "tty": '503-823-6868',
```

Comment:
Hmm. We should probably remove the references to TTY in the component as well: `web/themes/custom/cloudy/src/components/contact-item/contact-item.twig`.

Reply:
Wouldn't that break the contact field?

![image](https://user-images.githubusercontent.com/1520776/118199488-1c272e80-b408-11eb-9e92-2d426cdcbb61.png)

Reply:
It would. Good catch! You can ignore this one. 😄